lib/rototiller/utilities/block_syntax_object.rb

```diff
@@ -0,0 +1,12 @@
+module Rototiller
+  class Block_syntax
```

Comment:
maybe should remain just a module, then can be included into the Classes that use it?
feels more like a type of functionality than something to compose-in (shrug)